third_party/terraform/tests/resource_compute_autoscaler_test.go.erb

```diff
@@ -342,3 +369,38 @@ resource "google_compute_autoscaler" "foobar" {
 }
 `, autoscalerName)
 }
+
+<% unless version == 'ga' -%>
+func testAccComputeAutoscaler_scalingSchedule(itName, tpName, igmName, autoscalerName string) string {
+	return testAccComputeAutoscaler_scaffolding(itName, tpName, igmName) + fmt.Sprintf(`
+resource "google_compute_autoscaler" "foobar" {
+  description = "Resource created for Terraform acceptance testing"
+  name        = "%s"
+  zone        = "us-central1-a"
+  target      = google_compute_instance_group_manager.foobar.self_link
+  autoscaling_policy {
+    max_replicas    = 10
+    min_replicas    = 1
+    cooldown_period = 60
+    cpu_utilization {
+      target = 0.5
+    }
+    scale_down_control {
+      max_scaled_down_replicas {
+        percent = 80
+      }
+      time_window_sec = 300
+    }
+    scaling_schedules {
```

Comment:
Can you include an example with multiple schedules?

Reply:
fixed